From 5e0cf3d936284170b3b13c9e0e257cf3fc6d3664 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 26 Nov 2018 08:15:54 +0900
Subject: [PATCH 455/909] arm64: dts: renesas: r8a77990-ebisu: sort subnodes of
 root node

commit f0fc4fc5aa859da42f14e6b98cfa52e0e827ae4d from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch sorts the subnodes of the root node to improve maintainability.
The sort has been done alphabetically with the node name as the key.

This patch does not include functional changes.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index f342dd85b152..4aaa0080d76f 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -87,6 +87,15 @@
 		};
 	};
 
+	reg_3p3v: regulator1 {
+		compatible = "regulator-fixed";
+		regulator-name = "fixed-3.3V";
+		regulator-min-microvolt = <3300000>;
+		regulator-max-microvolt = <3300000>;
+		regulator-boot-on;
+		regulator-always-on;
+	};
+
 	vga {
 		compatible = "vga-connector";
 
@@ -119,15 +128,6 @@
 		};
 	};
 
-	reg_3p3v: regulator1 {
-		compatible = "regulator-fixed";
-		regulator-name = "fixed-3.3V";
-		regulator-min-microvolt = <3300000>;
-		regulator-max-microvolt = <3300000>;
-		regulator-boot-on;
-		regulator-always-on;
-	};
-
 	x13_clk: x13 {
 		compatible = "fixed-clock";
 		#clock-cells = <0>;
-- 
2.17.1

